src/domain/auth/README.md

```diff
@@ -0,0 +1,52 @@
+# HDS LoginProvider configurations
+
+Examples of the configurations for a local Tunnistamo and the Keycloak in the test environment.
+
+## Tunnistamo (local)
+
+```json
+{
+  "userManagerSettings": {
+    "authority": "http://tunnistamo-backend:8000/",
+    "client_id": "https://api.hel.fi/auth/kukkuu-ui",
+    "scope": "openid profile https://api.hel.fi/auth/kukkuu",
+    "redirect_uri": "http://localhost:3000/callback",
+    "silent_redirect_uri": "http://localhost:3000/silent_renew.html",
+    "post_logout_redirect_uri": "http://localhost:3000/"
+  },
+  "debug": false,
+  "sessionPollerSettings": { "pollIntervalInMs": 10000 },
+  "apiTokensClientSettings": {
+    "url": "http://tunnistamo-backend:8000/api-tokens/",
+    "maxRetries": 10,
+    "retryInterval": 1000
+  }
+}
+```
+
+# Keycloak (test environment)
+
+```json
+{
+  "userManagerSettings": {
+    "authority": "https://tunnistus.test.hel.ninja/auth/realms/helsinki-tunnistus/",
+    "client_id": "kukkuu-ui-dev",
```

Comment:
This (i.e. client_id="kukkuu-ui-dev") is for review/development environments according to the [Palveluintegraatio document](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/KAN/pages/8679030785/Palveluintegraatio+HP+-+Kukkuu+Kulttuurin+kummilapset), not test environment. So maybe switching to "kukkuu-ui-test" or renaming "test environment" would be clearer.

Reply:
Isn't "kukkuu-ui-test" the client ID used for test environment? At least according to the [Palveluintegraatio document](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/KAN/pages/8679030785/Palveluintegraatio+HP+-+Kukkuu+Kulttuurin+kummilapset).

Reply:
Yes, but this is a local machine configuration when the local kukkuu-ui is connected to the test-environment. So, a new paragraph really is needed. :) 

The kukkuu-ui-dev is the only one that has the localhost-addresses set in the callback urls. The test-profile is configured so that the dev-client can use it.

I have already tested this configuration that it works: this is the actual configuration that I was running the local client.

Reply:
It's a shame that we cannot see the actual configuration anyhow -- we always need to trust in 2nd party and in the SERvice integration document, that does not actualyl tell the configuration related to the client ids, scopes nor the callback urls.

Reply:
Changed